src/GraphControl/Control/Grapher.cpp

```diff
@@ -392,44 +389,42 @@ namespace GraphControl
                 int numValidEquations = 0;
                 for (Equation ^ eq : validEqs)
                 {
-                    if (numValidEquations++ > 0)
+                    if (eq->IsEnabled)
```

Comment:
What are the pros and cons of regraphing the whole graph vs setting the color for that equation as transparent? Would it be a lot of extra work? The one thing that comes to mind is the trace pop-up, but I think it would make the performance of this feature better.

Reply:
I'll request the API.

Reply:
After I merge your refactor PR, I'll look at just making the line transparent until we get the API. I the meantime, I have a new event for when the line is enable/disabled in preparation for a new api that can be called.